main.py

```diff
@@ -1658,3 +1660,43 @@ def objective(trial):
 plt.ylabel("Residuals")
 plt.title("ARima Model - Testing Data Residual Plot")
 plt.show()
+
+
+###Sentiment Analysis using DL
+
+# Function to seperate text and label
+def load_dataset(file_path, num_samples):
+    df = pd.read_csv(file_path, usecols=[3, 4], nrows=num_samples)
+    df.columns = ['review', 'rating']
+
+    text = df['review'].tolist()
+    text = [str(t).encode('ascii', 'replace') for t in text]
+    text = np.array(text, dtype=object)[:]
+    
+    labels = df['rating'].tolist()
+    labels = [1 if i>=7 else 0 if i>=5 else -1 for i in labels]
+    labels = np.array(pd.get_dummies(labels), dtype=int)[:] 
+
+    return labels, text
+    
+#Split into train and test dataset    
+tmp_labels, tmp_text = load_dataset('datasets\drugsComTrain_raw.tsv', 568454)
+test_labels, test_text = load_dataset('datasets\drugsComTest_raw.tsv',500000)
+
```

Comment:
Why to load and split the dataset again? This process is already done before @Garvit414

Reply:
Actually I have to split dataset into Text and labels. So firstly, I have make function for that  and after that I have call it for splitting.

Reply:
Ohkk fine.